code/modules/integrated_electronics/subtypes/lists.dm

```diff
@@ -84,6 +84,7 @@
 		"on failure" = IC_PINTYPE_PULSE_OUT,
 		)
 	icon_state = "addition"
+	complexity = 6
```

Comment:
Why 6?

Reply:
The process is comparable to the advanced list locator, which uses 6 complexity. Considering how much list processing is occuring in this one, it seemed appropriate, but do you think it should be higher?

Reply:
I think search should be lower

Reply:
I'll lower search to 2, then.